### PR TITLE
feat: Bump apisix-dashboard helm chart

### PR DESCRIPTION
**Pull request by hgranillo**
I'm opening this PR a second time because last time I made my commit on my forked master branch instead of in a feat branch.

Hello dear APISIX Helm chart maintainers! 

As a apisix user I would like to have the latest changes in the apisix-dashboard helm chart. 
I'm particularly interested in this fix https://github.com/apache/apisix-helm-chart/pull/323 for the ingressClassName field, I took the liberty of just bumping the version of the chart to `0.7.0`

Please let me know if further changes are required to release this chart, I will be happy to contribute!

ps: I believe this project would benefit of having something like helm-docs (https://github.com/norwoodj/helm-docs)

### CODE REVIEW

**Comment by tao12345666333** [comment]
yes. xref： #334

File: charts/apisix-dashboard/Chart.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.6.0
+version: 0.7.0
```
Comment (tao12345666333) [inline charts/apisix-dashboard/Chart.yaml:34]: Please use 0.6.1

In this project, 0.7.0 will only be used when the app releases a new version

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

File: charts/apisix-dashboard/Chart.yaml (hunk 1 above)
Comment (hgranillo) [reply charts/apisix-dashboard/Chart.yaml]: Ok. i've set the version to 0.6.1 instead. thanks!

**Review by tao12345666333 — APPROVED** [review]

**Comment by tao12345666333** [comment]
wait others approved

**Review by juzhiyuan — APPROVED** [review]

**Comment by juzhiyuan** [comment]
Thanks!

**Pull request merged**